guide/ops/locations/customizing_location_ports.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Customizing Location Ports
+layout: website-normal
+---
+
+## Opening ports during startup.
+
+Use the `org.apache.brooklyn.location.jclouds.networking.SharedLocationSecurityGroupCustomizer` location customizer.
+Example:
+
+    name: ports @ AWS
+    location:
+      jclouds:aws-ec2:us-west-2:
+        customizers:
+        - $brooklyn:object:
+            type: org.apache.brooklyn.location.jclouds.networking.SharedLocationSecurityGroupCustomizer
+            object.fields: {tcpPortRanges: ["900-910", "915", "22"], udpPortRanges: ["100","200-300"], cidr: "82.40.153.101/24"}
+        - $brooklyn:object:
+            type: org.apache.brooklyn.location.jclouds.networking.SharedLocationSecurityGroupCustomizer
+            object.fields: {tcpPortRanges: ["900-910", "915", "22"], udpPortRanges: ["100","200-300"], cidr: "10.40.153.101/16"}
+    services:
+    - type: org.apache.brooklyn.entity.software.base.EmptySoftwareProcess
+
+Make sure that you have rule which makes port 22 accessible from Apache Brooklyn.
+
+## Opening ports during runtime.
+
+Example configuration in effector
```

Comment:
Need some additional explanation to say what this does - that it adds an effector with a given name, etc.